Packs/ContentManagement/Scripts/SuggestBranchName/SuggestBranchName.py

```diff
@@ -4,6 +4,22 @@
 ATTEMPS = 10
 
 
+def find_available_branch_azure_devops(pack_name: str, command_get_branch: str):
+    response = demisto.executeCommand(command_get_branch, args={})
+    branch_name = pack_name
+    for i in range(ATTEMPS):
+        branch_exists = False
+        if i > 0:
+            branch_name = f'{pack_name}_{i}'
+        for branch in response[0].get("Contents", {}).get("value", []):
```

Comment:
please keep `response[0].get("Contents", {}).get("value", [])` in a variable called `existing_branches` at the beginning of this method, easier to understand what's going on